compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -2476,8 +2496,14 @@ fn add_native_libs_from_crate(
                 }
             }
             NativeLibKind::Unspecified => {
-                if link_dynamic {
-                    cmd.link_dylib(name, verbatim, true);
+                if prefer_link_static {
```

Comment:
```suggestion
                if !link_output_kind.can_link_dylib() {
```
Nit: it's only used once now.